### PR TITLE
[WIP] Formatting

### PR DESCRIPTION
**Pull request by CodaFi**
Leaving this here so we don't accidentally work on the same thing.

### CODE REVIEW

**Comment by pthariensflame** [comment]
:)

**Comment by CodaFi** [comment]
Ready for review :eyes:

File: Aquifer/NSFileHandleExt.m [hunk 1]
```diff
@@ -15,129 +15,140 @@
 #define BUFFER_SIZE 256
 
 in_addr_t getipaddr(const char *host) {
-    /* let's look it up */
```
Comment (CodaFi) [inline Aquifer/NSFileHandleExt.m:18]: Good god, what is this stuff?  I'm going to see what I can do about this weirdness.

Comment (pthariensflame) [inline Aquifer/NSFileHandleExt.m:18]: Please do!  :O

**Comment by pthariensflame** [comment]
This looks good to me so far.  I may end up going through the codebase after this is merged just to catch any subtle formatting errors, but for current purposes this is fine.

**Comment by CodaFi** [comment]
I'm done for now, then.

Merge when ready.

**Comment by pthariensflame** [comment]
:boat:, then!

**Pull request merged**